<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPembelian</class>
 <widget class="QWidget" name="FormPembelian">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>29</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="kode_barangLabel">
      <property name="text">
       <string>Kode Barang</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="kode_barangLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="kode_supplierLabel">
      <property name="text">
       <string>Kode Supplier</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="kode_supplierLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="no_fakturLabel">
      <property name="text">
       <string>No Faktur</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="no_fakturLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="tgl_fakturLabel">
      <property name="text">
       <string>Tanggal Faktur</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="harga_jualLabel">
      <property name="text">
       <string>Harga Jual</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="harga_jualLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="harga_beliLabel">
      <property name="text">
       <string>Harga Beli</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="harga_beliLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="banyak_keluarLabel">
      <property name="text">
       <string>Banyak Keluar</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="banyak_keluarLineEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QDateEdit" name="tgl_fakturDateEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>SImpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QTableView" name="tablePembelian">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>461</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
